distributed/locket.py

```diff
@@ -0,0 +1,198 @@
+# Vendored up-to-date copy of locket.py
+# Based on https://github.com/mwilliamson/locket.py/pull/8
```

Comment:
I'm curious if the locket maintainers would be willing to push a release quickly so that we could avoid vendoring this project.

cc @mwilliamson @eryksun

Fine either way, just thought I'd ask

Reply:
I'm a bit busy at the moment and probably don't have time to review the pull request, so probably not I'm afraid.